test/caesium/crypto/generichash_test.clj

```diff
@@ -33,3 +33,21 @@
       (is (array-eq (apply blake2b args)
                     empty-string-digest)
           (str "args: " args)))))
+
+(def empty-string (.getBytes ""))
+
+(def sha256-message (.getBytes "My Bonnie lies over the ocean, my Bonnie lies over the sea"))
+(def sha256-digest (unhexify "d281d10296b7bde20df3f3f4a6d1bdb513f4aa4ccb0048c7b2f7f5786b4bcb77"))
+(def sha256-empty-string-digest (unhexify "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"))
+
+(def sha512-message (.getBytes "My Bonnie lies over the ocean, Oh bring back my Bonnie to me"))
+(def sha512-digest (unhexify "2823e0373001b5f3aa6db57d07bc588324917fc221dd27975613942d7f2e19bf444654c8b9f4f9cb908ef15f2304522e60e9ced3fdec66e34bc2afb52be6ad1c"))
+(def sha512-empty-string-digest (unhexify "cf83e1357eefb8bdf1542850d66d8007d620e4050b5715dc83f4a921d36ce9ce47d0d13c5d85f2b0ff8318d2877eec2f63b931bd47417a81a538327af927da3e"))
```

Comment:
These are test vectors taken from https://github.com/abstractj/kalium/commit/e0e87455e8f791a1553bdaabf21c3acb3ca310d6 .

Is that a bad idea? I guess it means the test shows it is wrapping kalium correctly, but not that the functionality is actually correct.